automation/vars/default.yaml

```diff
@@ -0,0 +1,46 @@
+---
+vas:
+  hci:
+    repo: https://github.com/openstack-k8s-operators/architecture.git
+    branch: main
+    stages:
+    - path: examples/va/hci/control-plane/nncp
+      validation: "oc wait nncp --for jsonpath='{.status.conditions[0].reason}'=SuccessfullyConfigured --timeout=60" 
```

Comment:
We may want to ensure we pass the `-n <namespace>` in the validation, to be sure we're consuming the right namespace. Also, might be good to make it a list? So that we may iterate on it and get multiple validations. Any thoughts?

Reply:
Adding the namespace seems prudent.  Using a list instead also seems like a good idea.